lib/libUPnP/Platinum/Source/Devices/MediaServer/PltMediaItem.h

```diff
@@ -131,6 +131,7 @@ typedef struct {
     NPT_String toc;
     NPT_String user_annotation; //TODO: can be multiple
     NPT_UInt32 last_position;
+    NPT_String last_playerstate;
```

Comment:
Please put all Kodi-specific properties into the `PLT_XbmcInfo` structure because the other ones in `PLT_MiscInfo` are defined in the UPnP spec.